proposals/0029-cooperative-vector.md

```diff
@@ -506,17 +508,12 @@ Non-"Packed" type interpretations are used to request arithmetic conversions.
 Input type must be a 32-bit or 16-bit scalar integer or a 32-bit or 16-bit
 float. Integer to integer conversion saturates, float to float conversion is
 implementation dependent and preserves the value as accurately as possible.
-Float to integer conversion is RTNE and saturating. Integer to float conversion
-is RTNE.
-
-> TODO: These rules make sense for NN applications but diverge from HLSL
-> conversion rules
-> [here](https://microsoft.github.io/hlsl-specs/specs/hlsl.html#Conv).
+Float to integer conversion is RTNE (round to nearest even) and saturating.
+Integer to float conversion is RTNE. Given the diverse hardware used in
```

Comment:
For integer to float, what is the overflow behavior when the integer value is out-of-range of the selected float value?  Is it saturating, or does it generate +/-INF?  Or is it implementation defined?